blueprint/core/blueprint_AppHarness.cpp

```diff
@@ -1,85 +1,79 @@
-/*
```

Comment:
Noticed I removed these comments as they don't provide any value whatsoever (a `git log` or visit to GitHub will show any and all of this).

Realistcally, a top-of-the-file comment header license thing should be applied to all of your module's code files at some point.

Reply:
They really bugged me! I think thats an Xcode or Projucer generation thing ...

Reply:
Yeah, it definitely is.